manifests/components/nginx-ingress.jsonnet

```diff
@@ -158,6 +170,17 @@ local NGNIX_INGRESS_IMAGE = (import "images.json")["nginx-ingress-controller"];
           serviceAccountName: $.serviceAccount.metadata.name,
           //hostNetwork: true, // access real source IPs, IPv6, etc
           terminationGracePeriodSeconds: 60,
+          affinity+: {
```

Comment:
A comment that describes what this does would be very helpful.

Reply:
Since it's something we're likely to use a few times, I moved it into a more self-descriptive `utils.weakNodeDiversity`.

For anyone reading the PR archives, the relevant upstream anti-affinity docs are https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity